recipes/glueviz/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "0.7.3" %}
+
+package:
+  name: glueviz
+  version: {{version}}
+
+source:
+  fn: glueviz-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/6d/03/cbef6e6f80112dfc4edb4f1d6a419ea9b159511f0828b94f1bfd730f7cdb/glueviz-{{version}}.tar.gz
```

Comment:
Can you use https://pypi.io/packages/source/g/glueviz/glueviz-0.7.3.tar.gz instead?